app/code/Ometria/Core/Block/Head.php

```diff
@@ -103,10 +103,20 @@ public function getAPIKey()
      */
     public function isUnivarEnabled()
     {
```

Comment:
I have 2 questions/comments:

1. will the change of how this function behave, have impact on https://github.com/Ometria/magento2-extension/blob/f15ee81c1b2d1ed834d54fd59dddcb1b84ff3067/app/code/Ometria/Core/Helper/Cookiechannel.php#L44 ?
2. Can we have a better name now then `isUnivarEnabled` as this is doing now more then just
```php
public function isUnivarEnabled()
    {
        return $this->scopeConfig->getValue('ometria/advanced/univar');
    }
```
like before

maybe we should have 2 functions:
`isUnivarEnabled()` and `isCheckout()`

Reply:
@mkuzdowicz yes this can be two separate functions to be used as required in the different context. I think the misleading element here is the use of isUnivarEnabled in the tracking context [as you have pointed out](https://github.com/Ometria/magento2-extension/pull/32/files/fb77368404ff73b516cf73c7f33bdbf65665e43d#r717822560).